File: src/main/java/com/szczerbap/rssreader/controller/AdminController.java
package com.szczerbap.rssreader.controller;

import com.szczerbap.rssreader.data.dto.AllUserDto;
import com.szczerbap.rssreader.service.UserService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.*;

import java.util.List;

@RestController
@RequestMapping("/api/admin")
public class AdminController {

    @Autowired
    UserService userService;

    //===============================================================
    @RequestMapping(method = RequestMethod.GET)
    public List<AllUserDto> getAllUsers(){

        return userService.getAllUsers();

    }
    //================================================================
    @RequestMapping(value="/{userName}",method = RequestMethod.DELETE)
    public void removeUser(@PathVariable String userName){
        System.out.println(userName);
        userService.removeUser(userService.getByUserName(userName));
    }
    //================================================================
}

File: src/main/webapp/static/app/controller/AdminController.js
var app = angular.module('app')

app.controller('AdminController',function($http){

    var vm=this;

    getUsersList();
    function getUsersList() {

        $http({
            method:'GET',
            url:'/api/admin'
        }).then(function succes(response) {
            vm.users=response.data;
        },function error(response) {

        });
    };

    vm.removeUser=function (userName) {
        console.log(userName);
        $http({
            method:'DELETE',
            url:'/api/admin/'+userName
        }).then(function succes(response) {
            getUsersList();

        },function error(response) {

        });

    };


});